build-tools/automation/build.linux.groovy

```diff
@@ -3,7 +3,7 @@
 
 def XADir = "xamarin-android"
 def buildTarget = 'jenkins'
-def chRootPackages = 'xvfb xauth mono-devel autoconf automake build-essential vim-common p7zip-full cmake gettext libtool libgdk-pixbuf2.0-dev intltool pkg-config ruby scons wget xz-utils git nuget ca-certificates-mono clang g++-mingw-w64 gcc-mingw-w64 libzip-dev zulu-8 unzip lib32stdc++6 lib32z1 libtinfo-dev:i386 linux-libc-dev:i386 zlib1g-dev:i386 gcc-multilib g++-multilib referenceassemblies-pcl zip fsharp psmisc libz-mingw-w64-dev msbuild mono-csharp-shell devscripts fakeroot debhelper libsqlite3-dev sqlite3 libc++-dev cli-common-dev curl libncurses-dev'
+def chRootPackages = 'ant xvfb xauth mono-devel autoconf automake build-essential vim-common p7zip-full cmake gettext libtool libtool-bin libgdk-pixbuf2.0-dev intltool pkg-config ruby scons wget xz-utils git nuget ca-certificates-mono clang g++-mingw-w64 gcc-mingw-w64 libzip-dev zulu-8 unzip lib32stdc++6 lib32z1 libtinfo-dev:i386 linux-libc-dev:i386 zlib1g-dev:i386 gcc-multilib g++-multilib referenceassemblies-pcl zip fsharp psmisc libz-mingw-w64-dev msbuild mono-csharp-shell devscripts fakeroot debhelper libsqlite3-dev sqlite3 libc++-dev cli-common-dev curl libncurses-dev ninja-build'
```

Comment:
This needs rebasing atop 9887642623f6cea964e8cfe302bca83f6387d068.

Reply:
I know, I'm waiting for the current build to finish